src/Sulu/Component/PHPCR/PathCleanup.php

```diff
@@ -69,30 +81,30 @@ public function cleanup($dirty, $languageCode = null)
                 $dirty = str_replace($key, $value, $dirty);
             }
         }
-
-        $clean = strtolower($dirty);
-
-        // Inspired by ZOOLU
-        // delete problematic characters
-        $clean = str_replace('%2F', '/', urlencode(preg_replace('/([^A-za-z0-9\s\-_\/])/', '', $clean)));
-
         // replace multiple dash with one
-        $clean = preg_replace('/([-]+)/', '-', $clean);
+        $dirty = preg_replace('/([-]+)/', '-', $dirty);
```

Comment:
i think we should apply these filters after applying the slugger. otherwise we might sill get one of these things after applying the `implode`

Reply:
Will add tests for these

Reply:
Not a good idea either. Because it will explode multiple dashes and create empty array that will implode later and introduce same multiple dashes.

Reply:
but if we apply the filters after the `implode` calls the multiple slashes will be removed, right? 🤔